kboniadi/Lets_Play_Football
# Feat/admin utils

ok this is a pretty massive update... this probably should have been like three branches but oh well. Some notes regarding testing:

- _completed features are add, edit, and delete for souvenir's and just edit for stadium information_

### Test cases
_souvenir table_
1. click the add button and add a new souvenir item for a team (make sure to error check heavily!)
2. edit the price for any souvenir item (error check this as well)
3. delete multiple souvenir items

**note:** don't forget to verify that these changes are represented in the database

_information table_
1. There is no add'ing feature for this table, so make sure that the "Add" button remains greyed out at all times
2. when a row is selected, the contents should be displayed in the line-edits (make sure to heavily error check these inputs!)
3. Verify that everything is intuitive and makes sense in the grand scope of the logic
4. I was experimenting with the use of a status bar on the bottom of the screen, this is visible when double clicking on the editable tables. (method definition for this is in mainwindow for the time being)
